CMakeLists.txt

```diff
@@ -88,7 +86,6 @@ if(MEGA)
 	set(LOVE_LINK_DIRS ${SDL_LINK_DIR})
 
 	set(LOVE_LINK_LIBRARIES
-		${OPENGL_gl_LIBRARY}

```

Comment:
This change doesn't break anything?

Reply:
Quick test with my laptop, LOVE launches fine without it. Not sure about Linux though, but I think autotools is preferred there.